pkgs/top-level/perl-packages.nix

```diff
@@ -1619,6 +1619,7 @@ let
       homepage = "https://github.com/mongodb-labs/mongo-perl-bson-xs";
       description = "XS implementation of MongoDB's BSON serialization (EOL)";
       license = lib.licenses.asl20;
+      platforms = lib.platforms.linux; # configure phase fails with "ld: unknown option: -mmacosx-version-min=10.12"
```

Comment:
Would setting `export LD=$CC` environment work for this?  I reckon that's how we do on linker failures like this one on Darwin.

Reply:
Tried it, didn't work